<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionDraw_point"/>
    <addaction name="actionDraw_polyline"/>
    <addaction name="actionDraw_polygon"/>
    <addaction name="separator"/>
    <addaction name="actionMove_figure"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDraw_point"/>
   <addaction name="actionDraw_polyline"/>
   <addaction name="actionDraw_polygon"/>
   <addaction name="separator"/>
   <addaction name="actionMove_figure"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/settings.png</normaloff>:/Images/Images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDraw_point">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/pointer_4106.png</normaloff>:/Images/Images/pointer_4106.png</iconset>
   </property>
   <property name="text">
    <string>Draw point</string>
   </property>
  </action>
  <action name="actionDraw_polyline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/polyline.png</normaloff>:/Images/Images/polyline.png</iconset>
   </property>
   <property name="text">
    <string>Draw polyline</string>
   </property>
  </action>
  <action name="actionDraw_polygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/polygon.png</normaloff>:/Images/Images/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Draw polygon</string>
   </property>
  </action>
  <action name="actionMove_figure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/move.png</normaloff>:/Images/Images/move.png</iconset>
   </property>
   <property name="text">
    <string>Move figure</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/clear.png</normaloff>:/Images/Images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
